### PR TITLE
Fix homepage scroll bounties

### PR DESCRIPTION
**Pull request by Jeezman**
closes https://github.com/stakwork/sphinx-tribes/issues/984

## Screenshot
https://www.loom.com/share/9b64b16cea144bdcbe369470768dbff3

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
LGTM! Let's get a code review. Thanks.

**Comment by ecurrencyhodler** [comment]
@Jeezman follow up question. Is this enabled on the org page as well?

**Comment by Jeezman** [comment]
> @Jeezman follow up question. Is this enabled on the org page as well?

@ecurrencyhodler yes it is

**Comment by kevkevinpal** [comment]
@Jeezman can you fix the merge conflict I merged your earlier PR

github.com/stakwork/sphinx-tribes/pull/1016

**Comment by Jeezman** [comment]
@kevkevinpal currently AFK. I'd get to it as soon as I can

**Comment by Jeezman** [comment]
@kevkevinpal fixed conflicts. cc @ecurrencyhodler

File: frontend/app/src/pages/tickets/TicketModalPage.tsx [hunk 1]
```diff
@@ -72,42 +79,26 @@ export const TicketModalPage = observer(({ setConnectPerson }: Props) => {
     history.goBack();
   };
 
+  const directionHandler = (person: any, body: any) => {
+    if (person && body) {
+      if (bountyId) {
```
[the remark this answers is not in the document]
Comment (kevkevinpal) [reply frontend/app/src/pages/tickets/TicketModalPage.tsx:84]: I guess this isnt an important fix right now

**Pull request merged**